src/components/Book.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+import Image from '../images/progressCircle.png';
+
+const buttonStyle = {
+  backgroundColor: '#fff',
+  color: '#0290ff',
+  border: 'none',
+  cursor: 'pointer',
+  width: 'auto',
+};
+
+const updateButtonStyle = {
+  width: '15rem',
+  height: '1rem',
+  fontSize: '1rem',
+  marginTop: '2rem',
+};
+
+const containerStyle = {
+  display: 'flex',
+  justifyContent: 'space-around',
+  border: '2px solid #e8e8e8',
+  backgroundColor: '#fff',
+  padding: '2rem',
+  textTransform: 'none',
+};
+
+const buttonContainerStyle = {
+  display: 'flex',
+};
+
+const authorStyle = {
+  fontSize: '1.3rem',
+  color: '#4386bf',
+  margin: '1rem',
+};
+
+const titleStyle = {
+  fontSize: '2rem',
+  margin: '1rem',
+};
+
+const categoryStyle = {
+  fontSize: '1.3rem',
+  color: '#888888',
+  margin: '1rem',
+};
+
+const progressDescriptionStyle = {
+  color: '#888888',
+  fontSize: '1.3rem',
+  fontStyle: 'normal',
+  margin: '1rem',
+};
+
+const imageStyle = {
+  width: '10rem',
+  height: '10rem',
+};
+
+const progressFontStyle = {
+  fontSize: '3rem',
+  margin: '1rem',
+};
+
+const completedFontStyle = {
+  fontSize: '1.5rem',
+  margin: '0',
+};
+
+const generateRandomPercentile = () => {
+  const percent = '%';
+  return (Math.floor(Math.random() * 100) + 1) + percent;
+};
+
+const lineDivStyle = {
+  width: '0.125rem',
+  height: '10rem',
+  margin: '1.125rem 3.688rem 0 4.938rem',
+  backgroundColor: '#e8e8e8',
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const smallerLineDivStyle = {
+  width: '0.125rem',
+  height: '3rem',
+  backgroundColor: '#e8e8e8',
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const Book = (props) => {
+  const {
+    title,
+    author,
+    id,
+    category,
+  } = props;
+  const dispatch = useDispatch();
+  const remove = (bookToBeRemoved) => {
+    dispatch(removeBook(bookToBeRemoved));
+  };
+
+  return (
+    <div className="book" style={containerStyle}>
+      <div>
+        <h1 className="book_category" style={categoryStyle}>{category}</h1>
+        <h1 className="book_title" style={titleStyle}>{title}</h1>
```

Comment:
I cannot see the category of your project. Kindly add this to the book card. 

Original
<img width="250" alt="Screen Shot 2022-04-10 at 12 02 56 AM" src="https://user-images.githubusercontent.com/69011963/162588215-edeb5b1e-2afb-49bf-82b6-9fa66bf364df.png">


Your design
<img width="250" alt="Screen Shot 2022-04-10 at 12 02 41 AM" src="https://user-images.githubusercontent.com/69011963/162588217-c8fc0af9-9008-40f7-a731-1b4fe486633a.png">